### ISSUE TITLE
when training style.py, it got stuck, Help!

### ISSUE THREAD

**Issue by Uooga** [lengstrom/fast-style-transfer#188]
I try to train the model, but it shows as follows and then nothing happens.
I don't know how to solve it, waiting for your kindly solution, Thanks

2019-03-01 10:41:08.289612: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2799980000 Hz
2019-03-01 10:41:08.293372: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x2798100 executing computations on platform Host. Devices:
2019-03-01 10:41:08.293420: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
WARNING:tensorflow:From /home/xujia/.local/lib/python2.7/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From /home/xujia/.local/lib/python2.7/site-packages/tensorflow/python/ops/math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
UID:26

**Comment by JEF1056** [comment]
The first print messages don't occur until the first checkpoint is written. Be patient, and be happy that it didn't just print the UID and then say it's done when it didn't train whatsoever.

**Comment by DLove1204** [comment]
I have the same problems as like UID=80
so what now I have do is just wait ?

**Comment by JEF1056** [comment]
Yep, just wait, it'll print the iterations when it has finished one

**Comment by DLove1204** [comment]
thanks a lot!it's very helpful for me!appreciate it !!!!!!!!!!!!!!!

**Comment by taiftahmid** [comment]
I am facing the same issue. After waiting for 4 hours the training was still stuck! I was training style.py on google colab

**Comment by spot92** [comment]
I am running this on a 1080ti. Could I get some sort of print screen for what a successfully running train is? It has been sitting for awhile and I expected it to print of iterations as it went, but it is just sitting at UID.

**Comment by JEF1056** [comment]
@spot92 
Try changing `--checkpoint-iterations` to a lower value. It prints whenever it finishes that number of iterations

**Comment by spot92** [comment]
> @spot92
> Try changing `--checkpoint-iterations` to a lower value. It prints whenever it finishes that number of iterations

Thanks! How many iterations are in an epoch?
